mirage/scenarios/default.js

```diff
@@ -185,7 +185,8 @@ export default function(server) {
     organization,
     owner,
     slug: 'code-corps',
-    title: 'Code Corps'
+    title: 'Code Corps',
+    website: 'www.codecorps.org'
```

Comment:
I think we want this to be prefixed `https://` here, since that's what the server would do.